holidayextras/holidayextras.github.com
# Fixed styling of code elements inside list items

#### What does this PR do?
`code` elements inside `p` elements were styled with a grey background and border, but this was not applied to `code` elements inside lists (which are common, certainly in my own posts). I've updated the less file to target list items as well as paragraphs.

#### How can this be tested?
On a post, add a paragraph containing some content in a `code` element, then add a list and add another `code` example inside that. In both places, the `code` should be presented with the same grey background and border.

#### Screenshots / Screencast
<img width="808" alt="tinygrab screen shot 16-11-2015 17 12 29" src="https://cloud.githubusercontent.com/assets/12542645/11188969/619bf3de-8c85-11e5-9449-cb26dc50241f.png">
